worker/worker.go

```diff
@@ -631,6 +638,10 @@ func (w *Worker) LiveVideoToVideo(ctx context.Context, req GenLiveVideoToVideoJS
 	return resp.JSON200, nil
 }
 
+func (w *Worker) EnsureImageAvailable(ctx context.Context, pipeline string, modelID string) error {
```

Comment:
Where is it actually used? Is there any related go-livepeer PR?

Reply:
Ah sorry forgot to push https://github.com/livepeer/go-livepeer/pull/3279.